scripts/propagate-config.sh

```
@@ -0,0 +1,190 @@
+    echo "DRY_RUN: gh pr create --repo ${REPO} --head ${BRANCH} --base main --title \"${TITLE}\""
+    if [[ "$AUTO_MERGE" == "True" || "$AUTO_MERGE" == "true" ]]; then
+      # Per OMN-8838: always arm auto-merge via GraphQL enablePullRequestAutoMerge
+      # (never `gh pr merge --auto` — it silently picks the wrong method).
+      echo "DRY_RUN: gh api graphql enablePullRequestAutoMerge(mergeMethod: SQUASH) --auto ${REPO}#<pr>"
+    fi
+    continue
+  fi
+
+  # Live path (executed inside GitHub Actions runner with gh + git configured).
+  TMPDIR="$(mktemp -d)"
+  trap "rm -rf $TMPDIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)